pkg/cmd/pipeline/describe.go

```diff
@@ -36,6 +36,7 @@ import (
 
 const describeTemplate = `{{decorate "bold" "Name"}}:	{{ .PipelineName }}
 {{decorate "bold" "Namespace"}}:	{{ .Pipeline.Namespace }}
+{{decorate "bold" "Description"}}:	{{ .Pipeline.Spec.Description }}
```

Comment:
Should this be hidden if not specified? If the `Description` is empty, I think it can be excluded.

Reply:
done